### ISSUE TITLE
Beta 2 - Add accessibility text to all widgets

### ISSUE THREAD

**Issue by john-breton** [meanmedianmoge/zoia_lib#33]
The current UI is lacking accessibility text for most widgets. This should be rectified as accessibility text is necessary for some users to fully utilize the application. 
- Via the use of QtDesigner, this text can be added with ease.

**Comment by john-breton** [comment]
Accessibility text has been added throughout the application.
